src/FWorld.hpp

```diff
@@ -0,0 +1,35 @@
+#ifndef __FWORLD_HPP__
+#define __FWORLD_HPP__
+
+// FCharacter.cpp
+static const bool FACING_LEFT = false;
+static const float RESURRECTION_PERIOD = 3.f;
+static const float MOVEMENT_SPEED = 0.23f;
+static const float JUMPING_PERIOD = 0.4f; 
+static const FVector JUMP_VECTOR(0.f, -0.4f); // needs FVector loaded beforehand
+// AI constants skipped for now
```

Comment:
Yes I don't think they should be here anyway, they belong in the (future) AI class.